Build with Visual Studio 2015

When I generated Visual Studio (2015, 32-bit), solution using CMakeGui and then ran build of the 'amber' project, I got following compilation error:

> amber\samples\config_helper_vulkan.cc(643): error C2039: 'back_inserter': is not a member of 'std'

To fix this and successfully build working Amber, it sufficed to add this include:
` #include <iterator>`


> cl -version
Microsoft (R) C/C++ Optimizing Compiler Version 19.00.24215.1 for x86

--- a/samples/config_helper_vulkan.cc
+++ b/samples/config_helper_vulkan.cc
@@ -18,6 +18,7 @@
 #include <algorithm>
 #include <cstring>
 #include <iostream>
+#include <iterator>
 #include <set>
 #include <sstream>
 

